src/index.html

```diff
@@ -127,18 +172,18 @@ <h2 class="modal__header">New post</h2>
                 Please Fill out this field.
               </span>
             </label>
-            <label for="caption" class="modal__field">
+            <label for="name" class="modal__field">
               Caption
               <input
-                id="caption"
+                id="name"
```

Comment:
You shouldn't have changed `for="caption"` and `id="caption"` in the **card** form. Now the inputs doen't get **forcused** when I click on the label text. . Or you don't need `for` if the input is **inside** the `label` (it gets connected automatically)

It should be like this again:

```html

            <label for="caption" class="modal__field">
              Caption
              <input
                id="caption"
                type="text"
```